cellarium/ml/transforms/randomize.py

```diff
@@ -0,0 +1,35 @@
+# Copyright Contributors to the Cellarium project.
+# SPDX-License-Identifier: BSD-3-Clause
+
+
+import torch
+from torch import nn
+
+
+class Randomize(nn.Module):
+    """
+    Randomizely applies transform with probability p;
```

Comment:
Randomizely -> Randomly?

Reply:
Alternatively I removed the entire class, as implementation constraints dictated that I determine whether or not to apply transform in the transform itself (i.e. `p_apply`). In future, this logic could also be pulled out into some inheritable entity.